### PR TITLE
analyze: add new "pcrs" verb

### PR DESCRIPTION
**Pull request by poettering**
It shows the PCRs we know about along with their name strings and current values.

### CODE REVIEW

File: src/analyze/analyze.c [hunk 1]
```diff
@@ -234,6 +235,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "  inspect-elf FILE...        Parse and print ELF package metadata\n"
                "  malloc [D-BUS SERVICE...]  Dump malloc stats of a D-Bus service\n"
                "  fdstore SERVICE...         Show file descriptor store contents of service\n"
+               "  pcrs [PCR…]                Show TPM2 PCRs and their names\n"
```
Comment (YHNdnzj) [inline src/analyze/analyze.c]: Hmm, we use three dots everywhere else.

**Comment by bluca** [comment]
```
342s ../src/analyze/analyze-pcrs.c: In function ‘add_pcr_to_table’:
342s ../src/analyze/analyze-pcrs.c:78:21: error: ‘bufsize’ may be used uninitialized in this function [-Werror=maybe-uninitialized]
342s    78 |                 if (memeqbyte(0, buf, bufsize) ||
```

**Review by bluca — APPROVED** [review]
Please add a quick test to TEST-65-ANALYZE, doesn't need to do much just call with and without arguments, so that there's some basic sanitizer coverage

File: test/units/testsuite-65.sh [hunk 2]
```diff
@@ -838,6 +838,10 @@ systemd-analyze image-policy 'home=encrypted:usr=verity' 2>&1 | grep -q -e '^usr
 
 (! systemd-analyze image-policy 'doedel')
 
+# Output is very hard to predict, but let's run it for coverage anyway
+systemd-analyze pcrs
+systemd-analyze pcrs --json=pretty
+
```
Comment (bluca) [inline test/units/testsuite-65.sh:845]: can you also add one with a PCR number? so that side gets covered too

**Pull request merged**